charts/shoot-core/components/charts/vpn-shoot/templates/deployment.yaml

```diff
@@ -20,6 +20,7 @@ spec:
   replicas: 1
   strategy:
     rollingUpdate:
+    #is 1 and 0 okay
       maxSurge: 100%
       maxUnavailable: 0%
```

Comment:
Is 1 and 0 fine instead of 100% and 0% ? 
I saw `intStrOne  = intstr.FromInt(1)
	intStrZero = intstr.FromInt(0)`
in another component package for the same field. So I used that here.
If not, how to use it as %?

Reply:
Maybe just use what was in the Helm chart, so `100%`. Something like `intstr.FromString("100%")` should work, doesn't it?